src/main/java/com/ericsson/ei/subscription/EmailSender.java

```diff
@@ -95,14 +105,18 @@ public void sendMail(String receiver, String mapNotificationMessage, String emai
     }
 
     /**
-     * This method takes string of comma separated email addresses and return the Set of validated email
-     * addresses
+     * This method takes a string of comma separated email addresses and
+     * validates each of them before putting them in a Set of validated email
+     * addresses to return.
      *
-     * @param contents
+     * @param receivers
+     *     A string containing one or more comma separated email addresses
+     * @return emailAdd
+     * @throws SubscriptionValidationException
      */
-    public Set<String> extractEmails(String contents) throws SubscriptionValidationException {
+    public Set<String> extractAndValidateEmails(String receivers) throws SubscriptionValidationException {
```

Comment:
The emails are validated at creation of the subscription, validation here may be removed and it would simplify the code and remove the need to throw a SubscriptionValidationException.

Reply:
Will fix